src/shared/components/base-question-app.tsx

```diff
@@ -30,6 +30,7 @@ export interface IRuntimeQuestionComponentProps<IAuthoredState, IInteractiveStat
   interactiveState?: IInteractiveState | null,
   setInteractiveState?: (updateFunc: UpdateFunc<IInteractiveState>) => void;
   report?: boolean;
+  view?: string;
```

Comment:
This should be restricted to "standalone".  It is probably best to just explicitly put that here for now.

But if you wanted to delve into typescript stuff, you can probably find a way to pull the possible values out of the lara-interactive-client types.